.github/workflows/ci.yml

```diff
@@ -69,6 +70,7 @@ jobs:
 
           - { os: 'windows-latest', language: 'dotnet',  language_version: '8.0.x' }
           - { os: 'windows-latest', language: 'go',      language_version: '1.21'  }
+          - { os: 'windows-latest', language: 'rust',    language_version: '1.84'  }
```

Comment:
Let's do `language_version: 'stable'`, so that we automatically test newer Rust once it is released. This is the semantics we should ideally use for all other languages, but Rust makes it especially easy, because `rustup` is good!